provider/cluster/kube/client.go

```diff
@@ -62,23 +87,15 @@ func openKubeConfig(log log.Logger) (*rest.Config, error) {
 
 func (c *client) Deployments() ([]cluster.Deployment, error) {
 
-	// TODO: loop to more pages.
-	namespaces, err := c.kc.CoreV1().Namespaces().List(metav1.ListOptions{
-		LabelSelector: deploymentSelector().String(),
-	})
-
+	manifests, err := c.mc.AkashV1().Manifests(manifestNamespace).List(metav1.ListOptions{})
 	if err != nil {
 		return nil, err
 	}
 
 	var deployments []cluster.Deployment
 
-	for _, ns := range namespaces.Items {
-		obj, err := deploymentFromAnnotation(ns.Annotations)
-		if err != nil {
-			return nil, err
-		}
-		deployments = append(deployments, obj)
+	for _, manifest := range manifests.Items {
```

Comment:
Can you check to see if we're `manifest.LeaseID.Provider`?  `_run/multi` will be confused otherwise.  Alternative: make `manifestNamespace` a CLI parameter and string that through (`akash provider run --kube --manifest-ns us-west-1`)

Reply:
is passed by cli flag now